pkg/cli/admin/catalog/mirror.go

```diff
@@ -125,6 +127,7 @@ func NewMirrorCatalog(f kcmdutil.Factory, streams genericclioptions.IOStreams) *
 	flags.StringVar(&o.FileDir, "dir", o.FileDir, "The directory on disk that file:// images will be copied under.")
 	flags.StringVar(&o.FromFileDir, "from-dir", o.FromFileDir, "The directory on disk that file:// images will be read from. Overrides --dir")
 	flags.IntVar(&o.MaxPathComponents, "max-components", 2, "The maximum number of path components allowed in a destination mapping. Example: `quay.io/org/repo` has two path components.")
+	flags.StringVar(&o.IcspScope, "icsp-scope", "repository", "Scope of registry mirrors in imagecontentsourcepolicy file. Allowed values: repository, registry. Defaults to: repository")
```

Comment:
the default should be placed into the `NewMirrorCatalogOptions` https://github.com/openshift/oc/pull/485/files#diff-7db656e03ae09bb1778bb0a8cde7fd18R99

Reply:
ok, sorry for the confusion. Done now.